vkrunner/vr-execute.c

```diff
@@ -27,6 +27,7 @@
 
 #include "vr-execute.h"
 
+#include <assert.h>
```

Comment:
This header is not used anywhere in the rest of the file.

Reply:
Removed.